feature.js

```diff
@@ -204,18 +212,27 @@
     testAll : function() {
       var classes = " js";
       for (var test in this) {
-        if (test !== "testAll" && test !== "constructor" && this[test]) {
+        if (test !== "testAll" && test !== 'extend' && this[test]) {
```

Comment:
Maybe we should try be consistent which quotes we use. I’m fine with either one, but if we switch to single quotes, should update the whole library to follow the same style.

Reply:
Yeah, I was trying to keep my work atomic while opening a deluge of PRs over the last few days. This should be find after the ESLint work merges, but if not, I'll gladly resolve any conflicts.